documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+  types of feedback to give
+- Adopt more specific standards about what types of feedback must be addressed
+  before a proposal can be ratified for implementation
+- Give specificity in feedback expectations, both for who will participate in a
+  decision-making process and what responsibilities and tools they have for
+  participating
+- Reaffirm trust between contributors when evaluating proposals
+
+Historically, each of these have presented problems for the Openverse project as
+it has grown. Decision-making can drag on for months, proposal reviews get stuck
+in feedback cycles over issues that could be addressed later on, and there were
+few, if any, expectations around who would participate in a discussion or what
+the expectations of that participation was. Adopting the process described
+below, we hope, will address some of the underlying causes of these problems, in
+addition to the benefits described above.
+
+The process described below is heavily modified from the regular Sociocratic
+process. The differences come from two directions:
+
+- The needs of a largely asynchronously coordinated base of contributors
+- Internal feedback and iteration on the initial process, even after
+  modification for asynchrony
+
+> ## When to follow this process
+>
+> Whether this process is followed for any given proposal or discussion is left
+> entirely to the discretion of the people relevant to the discussion. We
+> believe that this process helps the project's contributors make expedient and
+> clear decisions. However, it should not become an obstacle that is followed
+> for every tiny decision. If a decision can efficiently be made without this
+> process, contributors are invited to do so.
+>
+> For the purposes of project planning, however, this process should be the
+> default and should be the strong preference, keeping in mind that it can be
+> made faster via [round shortcutting](#shortcutting-a-round) and the
+> [synchronous lightening process](#optional-synchronous-lightening-process).
+
+## Continuous improvement and exceptions to every rule
+
+**This process is not set in stone.** Nor is it a set of laws that must be
+followed at all times. This document uses prescriptive language filled with
+imperatives primarily to prevent the process itself from being filled with
+contingencies at every turn. However, as with every other aspect of our ever
+evolving processes, these are merely suggestions that we expect contributors to
+adhere to. They cannot cover every situation and undoubtedly there are
+exceptional situations where following these suggestions to the letter would be
+antithetical to the goals of the project and even of the process itself.
+
+If during your time as an Openverse contributor, you feel there are ways in
+which this process can be improved and iterated upon, please know that this
+process should be iterated upon and that suggestions for improvement are open
+from any contributor to the project.
```

Comment:
Another simplification suggestion, perhaps: `If you have ideas for improving the Openverse decision-making process, please share them. Suggestions are open to any contributor.`